### PR TITLE
add notifications settings tab

### LINKED ISSUE #4231: Subscribe to email notifications from Settings page

### PR DESCRIPTION
**Pull request by kdembler**
~**⚠️  Based on #4551 and #4561, review those first ⚠️**~

Adds notifications tab to settings

Resolves #4231

### CODE REVIEW

File: packages/ui/src/app/pages/Settings/SettingsNotificationsTab.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      return {
+        accounts: parameters.isLoggedIn
+          ? { active: { member: alice, balances: parameters.balance } }
+          : { list: [{ member: alice }] },
+
+        gql: {
+          queries: [
+            {
+              query: GetBackendMemberExistsDocument,
+              data: { memberExist: args.isRegistered },
```
Comment (thesan) [inline packages/ui/src/app/pages/Settings/SettingsNotificationsTab.stories.tsx:56]: Just to easily check these states:
```suggestion
              resolver: () => {
                data: { memberExist: args.isRegistered },
                isLoading: args.activeMemberExistBackendLoading,
                error: args.activeMemberExistBackendError,
              }
```

File: packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    await sendUpdateMemberMutation({
+      variables: {
+        email: dirtyFields.email ? data.email : undefined,
+        receiveEmails: dirtyFields.receiveEmailNotifications ? data.receiveEmailNotifications : undefined,
+      },
+    })
+    setShowSettingsUpdatedModal(true)
+  })
+
+  const loadingContent = 'Loading notification settings'
```
Comment (thesan) [inline packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx:108]:
```suggestion
  const loadingContent = <Loading text="Loading notification settings" />
```

File: packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  const errorContent = 'Error loading notification settings'
```
Comment (thesan) [inline packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx:109]:
```suggestion
  const errorContent = (
    <Warning
      title="Verify your email account"
      content="Error loading notification settings"
      isClosable={false}
    />
  )
```

**Review by thesan — CHANGES_REQUESTED** [review]
Really great PR :star_struck: 

Just 2 small changes for the error and loading state

File: packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx (hunk 2 above)
Comment (kdembler) [reply packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx]: I was planning to do that as part of issue with handling errors, but thanks for your suggestions, applied.

File: packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx (hunk 3 above)
Comment (kdembler) [reply packages/ui/src/app/pages/Settings/SettingsNotificationsTab.tsx]: Done

File: packages/ui/src/app/pages/Settings/SettingsNotificationsTab.stories.tsx (hunk 1 above)
Comment (kdembler) [reply packages/ui/src/app/pages/Settings/SettingsNotificationsTab.stories.tsx]: 👍

**Review by thesan — APPROVED** [review]
:100:

**Pull request merged**